### PR TITLE
Add clarity to HA vs. Hassio vs. HassOS question

### PR DESCRIPTION
**Pull request by ghost**
**Description:**
I think the current wording in this FAQ is incorrect.  It blurs the line between Hass.io and HassOS.  It also doesn't seperate Hass.io from HassOS.  I think the proposed changes adds some clarity to the discussion. 

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [X ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_faq/ha-vs-hassio.markdown [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
-title: "Home Assistant vs. Hass.io"
-description: "Home Assistant vs. Hass.io"
+title: "Home Assistant vs. Hass.io vs. HassOS"
+description: "Home Assistant vs. Hass.io vs. HassOS"
 ha_category: Installation
 ---
 
-Home Assistant is a Python program, in simple words. It can be run various operating system and provide the ability to track, control and automate your devices. When people talking about Home Assistant they usually refer to a standalone [installation method](/docs/installation/).
+Home Assistant is a Python program and is run via various [Installation Methods](https://www.home-assistant.io/docs/installation/).  The home assistant Python program is the core of the Home Assistant project that is used to observe, control and automate the devices in your home.
```
Comment (frenck) [inline source/_faq/ha-vs-hassio.markdown:7]: Besides adding a spelling error, I don't see any added value to this change.
I would suggest on reverting this part.

File: source/_faq/ha-vs-hassio.markdown [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
-[Hass.io](/hassio/) is a combination of Home Assistant and tools which allows one to run it easily on a Raspberry Pi and other platforms without setting up an operating system first. Hass.io is an all-in one-solution and has a management user interface that can be used from the Home Assistant frontend. This interface is not present in a standalone setup of Home Assistant.
+[Hass.io](https://www.home-assistant.io/hassio/) is a Docker-based system for managing your Home Assistant installation and related applications. Hass.io is an all-in one-solution and has a management user interface that can be used from the Home Assistant frontend. This interface is not present in a standalone setup of Home Assistant. Hass.io can also utilize [Add-ons](https://www.home-assistant.io/addons/) to extend the functionality around Home Assistant. Hass.io can be [installed on compatiable linux distribution](https://www.home-assistant.io/hassio/installation/#alternative-install-on-a-generic-linux-host) running docker or is part of the HassOS installation method.
```
Comment (frenck) [inline source/_faq/ha-vs-hassio.markdown:9]: This paragraph now contains a ton of information and is hard to read. Consider changing it to make it less wordy.

Besides that, I'm not sure if starting with `is a Docker-based` helps the new user. He/She might be instantly confused. It is probably better to move that later into the explanation.

`This interface is not present in a standalone setup of Home Assistant` -> Also not present in Docker-based systems or Hassbian. Maybe it is clearer to state it is only available on Hass.io installations?

`Hass.io can be installed on compatiable linux distribution running docker or is part of the HassOS installation method. `

The running Docker part might be confusing here. So what is HassOS in this case? It is explained lower, but "part of" sounds a bit weird to me in this context.

File: source/_faq/ha-vs-hassio.markdown [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+[Hass.io](https://www.home-assistant.io/hassio/) is a Docker-based system for managing your Home Assistant installation and related applications. Hass.io is an all-in one-solution and has a management user interface that can be used from the Home Assistant frontend. This interface is not present in a standalone setup of Home Assistant. Hass.io can also utilize [Add-ons](https://www.home-assistant.io/addons/) to extend the functionality around Home Assistant. Hass.io can be [installed on compatiable linux distribution](https://www.home-assistant.io/hassio/installation/#alternative-install-on-a-generic-linux-host) running docker or is part of the HassOS installation method.  
+
+[HassOS](https://github.com/home-assistant/hassos) is a Operating System specifcally deisgned to run Hass.io.  The whole system is optimized for the embedded system, security and allows for seamless updates of the underlying Operating System and Hass.io.
```
Comment (frenck) [inline source/_faq/ha-vs-hassio.markdown:11]:
```suggestion
[HassOS](https://github.com/home-assistant/hassos) is an Operating System specifically designed to run Hass.io.  The whole system is optimized for speed, security and allows for seamless, single-click, updates of the underlying HassOS Operating System, Hass.io, add-ons and Home Assistant itself.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there, @SilvrrGIT!

I've reviewed your PR and left some comments. Could you please take a look? 👍 

Furthermore, please change the target branch of this PR to `current`, since this changes our current documentation and does not rely on the next release. Rebase the PR onto `current` if needed.

Thanks 👍

**Comment by ghost** [comment]
Let me rework and resubmit this.

**Pull request closed without merging**